src/fundus/publishers/us/fox_news.py

```diff
@@ -14,7 +15,11 @@
 
 class FoxNewsParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector(".article-body > p")
+        _summary_selector = CSSSelector(".article-meta > h2")
+        _paragraph_selector = XPath(
+            "(//div[@class='article-body'] | //div[@class='article-body']/div[@class='paywall']) "
+            "/p[not(child::script) and text()]"
```

Comment:
```suggestion
            "//p[not(child::script) and text()]"
```
For some reason within fundus it needs the extra slash

Reply:
That's exactly what we don't want to have. Having the extra slash will select all `p` not only those following but also those  just being after `(//div[@class='article-body'] | //div[@class='article-body']/div[@class='paywall'])` in the DOM tree, collecting a lot of junk in the process.

Reply:
Ok, I didn't see that when testing. But without the extra slash, the "premium" articles plaintext comes back as None